homeassistant/config_entries.py

```diff
@@ -70,6 +70,8 @@
 
 SAVE_DELAY = 1
 
+T = TypeVar("T", bound="ConfigEntryState")
```

Comment:
```suggestion
_T = TypeVar("_T", bound="ConfigEntryState")
```
The TypeVar should only every be needed in the `ConfigEntryState` class.